packages/core/src/dataTypes/enumerate/getFinalize.ts

```diff
@@ -51,7 +95,7 @@ export function getFinalize(
     // Example : ?filter[age]=xyz ... xyz => throw "xyz is not a string";
     setORMQueryFilterValue: (queryValue: string): string | null => {
       if (["null", "undefined"].includes(queryValue)) {
-        if (control.allowNull !== false) {
+        if (control.allowNull) {
```

Comment:
are you sure we need to change this? (backend functions)
i think allowNull can be `undefined|boolean` so should be careful removing this check for explicit false

Reply:
That's fair